### DISCUSSION TITLE
Support for RKE1 upgrade

### DISCUSSION THREAD

**Discussion by MisderGAO** [rancher/system-upgrade-controller#258] [Q&A]
May I know if you are going to support RKE1 automated upgrade in the near future.
Thanks.

**Comment by brandond** [comment]
The SUC does not "support" any specific product. See the documentation, and examples directory for examples of using it to upgrade OS packages. You could probably come up with a plan that would upgrade your RKE1 cluster, but I am not aware of any plans for us to provide or support such a thing.
